app/javascript/src/navigation/components/Header.js

```diff
@@ -1,71 +1,176 @@
+                  </HeaderNavCenter>
+                </HeaderNav>
+              )
+            }}
+          </Sticky>
+        </DesktopHeader>
+        <MobileHeader>
+          <div>
+            <LogoLink to='/'>
+              <img src={logo} alt='Legislated' height='40' width='40' />
+            </LogoLink>
+            <LogoLink to='/'>
+              <span>LEGISLATED</span>
+            </LogoLink>
+          </div>
+          <MobileNav />
+        </MobileHeader>
       </div>
-    </div>
+    )
   }
 }
 
-const rules = stylesheet({
-  container: {
-    ...mixins.borders.low(['bottom']),
-    display: 'flex',
-    justifyContent: 'space-between',
-    alignItems: 'center',
-    height: 80,
-    paddingLeft: 30,
-    paddingRight: 30,
-    backgroundColor: alpha(colors.background, 0.95),
-    ...mixins.mobile({
-      height: 60,
-      paddingLeft: 15,
-      paddingRight: 15
-    })
-  },
-  logoLink: {
-    color: colors.black,
-    fontSize: 32,
-    textDecoration: 'none',
-    transition: 'color 0.25s',
-    ':hover': {
-      color: alpha(colors.black, 0.6)
-    },
-    '> img': {
-      verticalAlign: 'top'
-    },
-    '> span:last-child': {
-      ...mixins.fonts.bold,
-      marginLeft: 15,
-      letterSpacing: 5
-    },
-    ...mixins.mobile({
-      fontSize: 28
-    })
-  },
-  nav: {
-    ...mixins.borders.low(['left']),
-    flex: 1,
-    display: 'flex',
-    marginLeft: 15,
-    paddingLeft: 15,
-    ...mixins.mobile({
-      display: 'none'
-    })
+const DesktopHeader = styled.div`
+  @media (max-width: 700px) {
+    display: none;
+  }
+`
+
+const HeroLogo = styled.div`
+  height: 280;
+  background-color: white;
+  display: flex;
+  flex-flow: column nowrap;
+  justify-content: space-evenly;
+  align-items: center;
+
+  > a {
+    font-size: 64px;
+  }
+`
+
+const HeaderNav = styled.div`
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: ${props => props.isSticky ? 'space-between' : 'center'};
+  height: 85px;
+  background-color: white;
+  z-index: 2;
+`
+
+const StickyLogo = styled.div`
+  display: flex;
+  flex-flow: row nowrap;
+  padding-left: 27px;
+
+  > *:first-child {
+    margin-right: 15px;
+  }
+
+  > a {
+    font-size: 32px;
+  }
+`
+
+// Includes a hack to style the active links due to a specificity issue
+// causing the styles not to applied. This is not ideal as its tied to
+// the aria-current attribute
+const HeaderNavCenter = styled.div`
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: center;
+
+  > a {
+    flex: 0 1 auto;
+    padding: 0 27px;
+    text-transform: uppercase;
+    border-bottom: 7px solid white;
+    ${css(mixins.centerVertically)};
+    text-decoration: none;
+    color: ${colors.light};
+    ${css(mixins.fonts.light)};
+
+    &[aria-current="true"] {
+      color: ${colors.newPrimary};
+      border-bottom: 7px solid ${colors.newPrimary};  
+    }
+  }
+`
+
+const MobileHeader = styled.div`
+  flex-flow: row nowrap;
```

Comment:
i think `nowrap` is the default for `flex-wrap`, and `row` in the default for `flex-direction`. can these be removed for rows and replaced w/ `flex-direction: column` columns?

Reply:
yeah, for some reason, i thought the default for `flex-wrap` was `wrap`